src/components/LandingPage/Testmonials.js

```diff
@@ -0,0 +1,126 @@
+    <>
+      <div className={classes.div}>
+        <h2 className={classes.header}>
+        Have you faced adversity and want to learn tech skills?
+        </h2>
+        <p className={classes.text}>
+          If you are a person facing facing homelessness and housing precarity, we will work with you to give you the tech knowledge, skills and advice to help you achieve your personal, professional or entrepreneurial goals.
+        </p>
+        <ExpandingTextBox
+          className={classes.textBox}
+          mainText={
+            " Tiana grew up in foster care. Since aging out of the system at 18 they no longer have access to the support they once had, and have faced prejudice and mistreatment from past shelters for being transgender. Like many others in this situation they have faced discriminination, trust and confidence issues which were significant barriers in pursuing education. Tiana loves photography and is curious about web design so when they heard about Toronto Tech Mentoring through a friend at their shelter, they contacted the TTM team to learn more. The program volunteers helped them to navigate their goals, understand their learning style, and customize a website project specifically for them that would allow them to gain hands on experience while learning the basics of web design. They really enjoyed it, and after completing their project, their mentors helped them explore web design as a career path."
+          }
+          name={"Tiana"}
+        />
+        <TestimonialButton
+          buttonText={"Join our program"}
+          className={classes.button}
+        />
+      </div>
+      <div className={classes.div}>
+        <h2 className={classes.header}>
+          Are you interested in mentoring someone or helping us run this
+          program?
+        </h2>
+        <p className={classes.text}>
+          We come from different backgrounds (not all tech!) and build collaborative and supportive environments. We are always looking for more dedicated individuals to join our team!
+        </p>
+        <ExpandingTextBox
+          className={classes.textBox}
+          mainText={
+            "Enzo is a recent graduate with ambitions of being a Software Developer and getting his foot in the door in Toronto’s thriving tech industry. Since joining Toronto Tech Mentoring’s volunteer team, Enzo has been building his portfolio, developing his soft skills, and expanding his professional network through collaboration with other volunteer team members, including more senior developers, user researchers and other professionals —all while contributing to a cause he cares about!"
+          }
+          name={"Enzo"}
+        />
+        <TestimonialButton
+          buttonText={"Volunteer with us"}
+          className={classes.button}
+        />
+      </div>
+      <div className={classes.div}>
+        <h2 className={classes.header}>
+        Want to help us continue to provide this program, and expand our project?
```

Comment:
Text here needs to be changed:

"Are you part of an organization that works to help people who face adversity?"

Reply:
Also, there was a typo in figma, could you remove the second "facing" in line 50: 

![Screen Shot 2021-03-15 at 11 18 09 PM](https://user-images.githubusercontent.com/59675897/111251001-c4756e00-85e4-11eb-96a2-f697c9a8cd41.png)